redesign/src/styles/layout/_404.scss

```diff
@@ -0,0 +1,129 @@
+.main.error-page {
+    margin-bottom: 160px;
+}
+
+.button {
+    &--option {
+        border: 1px solid $grey;
+        border-radius: 4px;
+        box-sizing: border-box;
+        cursor: pointer;
+        user-select: none;
```

Comment:
Can you discuss why you've added this here?

Reply:
Sorry, which one or ones are we talking about?

Reply:
No worries! The `user-select` here and elsewhere -- can you add some comments about what it's accomplishing?